src/main/java/seedu/eduke8/topic/Topic.java

```diff
@@ -1,23 +1,30 @@
 package seedu.eduke8.topic;
 
-import seedu.eduke8.question.QuestionListInterface;
+import seedu.eduke8.common.Displayable;
+import seedu.eduke8.question.QuestionList;
 
-public class Topic implements TopicInterface {
-
-    private String topicName;
-    private QuestionListInterface allQuestions;
+public class Topic implements Displayable {
```

Comment:
Changed existing interfaces into classes under 2 broad interfaces, could make the code tidier